models/brats_mri_axial_slices_generative_diffusion/configs/train_diffusion.json

```diff
@@ -65,15 +65,15 @@
         "beta_start": 0.0015,
         "beta_end": 0.0195
     },
-    "inferer": {
-        "_target_": "generative.inferers.LatentDiffusionInferer",
-        "scheduler": "@noise_scheduler",
-        "scale_factor": "@scale_factor"
-    },
     "loss": {
         "_target_": "torch.nn.MSELoss"
     },
     "train": {
+        "inferer": {
```

Comment:
FYI @Can-Zhao , I changed to define "inferer" within "train" in order to align with:
https://github.com/Project-MONAI/MONAI/blob/858b86cf3af36e17efac8661533e3aa642dff5be/monai/bundle/properties.py#L66